packages/components/src/autocomplete/index.js

```diff
@@ -144,15 +141,150 @@ function getRange() {
 	return selection.rangeCount ? selection.getRangeAt( 0 ) : null;
 }
 
+const getAutoCompleterUI = ( autocompleter ) => {
+	const useItems = autocompleter.useItems
+		? autocompleter.useItems
+		: ( filterValue ) => {
```

Comment:
Are we confident now to document `useItems`, or is there a reason to wait?

Reply:
There are some potential things we may want to add, like a way to show alternative UIs (not just a flat list) but I do feel confident about useItems.